src/services/EdFi.DataManagementService.Api.Tests/EndpointsTests.cs

```diff
@@ -3,10 +3,10 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
+using System.Net;
 using FluentAssertions;
 using Microsoft.AspNetCore.Mvc.Testing;
 using NUnit.Framework;
-using System.Net;
 
 namespace EdFi.DataManagementService.Api.Tests
 {
```

Comment:
I'd be interested in seeing what csharpier does if you change this to the newer syntax: add a semicolon at the end of line 11, delete line 12 and the closing brace somewhere below, and then you're able to outdent the entire block to save some left side whitespace.

Reply:
It did the right thing.